### PR TITLE
Support #r @" " and #r """ """ strings

### PR DESCRIPTION
**Pull request by KevinRansom**
Disable response file support for kernel directives.  This ensures that @"" and """ """ are passed through to interactive.

Re-enable the tests for this.

### CODE REVIEW

File: Microsoft.DotNet.Interactive/Extensions/CommandLineBuilderExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+using System;
+using System.CommandLine;
+using System.CommandLine.Builder;
+
+namespace Microsoft.DotNet.Interactive.Extensions
+{
+    public static class CommandLineBuilderExtensions
+    {
+        public static CommandLineBuilder ResponseFile(
```
Comment (jonsequitur) [inline Microsoft.DotNet.Interactive/Extensions/CommandLineBuilderExtensions.cs]: This method already exists `System.CommandLine`:

https://github.com/dotnet/command-line-api/blob/0fd4b6381f62993c16414b00fcb4804302d04890/src/System.CommandLine/Builder/CommandLineBuilderExtensions.cs#L100-L106

Comment (KevinRansom) [reply Microsoft.DotNet.Interactive/Extensions/CommandLineBuilderExtensions.cs]: Jon … it's a property.  The helper method sets the property.

Comment (jonsequitur) [reply Microsoft.DotNet.Interactive/Extensions/CommandLineBuilderExtensions.cs]: The method in `System.CommandLine` is also setting the property and also returning the `CommandLineBuilder` instance. The implementation is identical (other than the null check).

**Comment by KevinRansom** [comment]
@colombod can we get this merged pls.

**Pull request merged**